sunpy/map/tests/test_plotting.py

```diff
@@ -167,7 +167,7 @@ def test_plot_masked_aia171_nowcsaxes(aia171_test_map_with_mask):
 
 @figure_test
 def test_plot_aia171_superpixel(aia171_test_map):
-    aia171_test_map.superpixel((9, 7) * u.pix, offset=(4, 4) * u.pix).plot()
+    aia171_test_map.superpixel((3, 2) * u.pix, offset=(4, 4) * u.pix).plot()
```

Comment:
I changed this to check that the changes looked fine on a slightly higher resolution map.